jmespath-core/src/main/java/io/burt/jmespath/node/ComparisonNode.java

```diff
@@ -4,8 +4,6 @@
 import io.burt.jmespath.Expression;
 import io.burt.jmespath.JmesPathType;
 
-import static io.burt.jmespath.node.Operator.*;
```

Comment:
I don't know, but it seems the fact that we are switching on an `Operator` type is enough for the compiler to know?